client/apps/Planning/PlanningListSubNav.tsx

```diff
@@ -51,11 +57,16 @@ class PlanningListSubNavComponent extends React.Component<IProps, IState> {
     sortFieldOptions: Array<{label: string, onSelect(): void}>
     container?: HTMLDivElement;
     resizeObserver: ResizeObserver;
+    userList: Array<{label: string, onSelect(): void}>
 
     constructor(props) {
         super(props);
+        this.state = {
```

Comment:
Rather than storing this locally, it would be better to pass in this search param from a higher up component, like we're already doing with the other search params in this component

Reply:
`state` is still being used. It is better to pass in the search param instead of storing it locally